src/yandex_cloud_ml_sdk/_files/domain.py

```diff
@@ -132,10 +153,11 @@ async def _list(
 
                 page_token_ = response.next_page_token
 
-
+@doc_from(BaseFiles)
```

Comment:
You need to add import 
```
from yandex_cloud_ml_sdk._utils.doc import doc_from
```
to make it work, otherwise as you can see, tests failed because of it